source/features/more-dropdown.tsx

```diff
@@ -29,35 +29,38 @@ async function init(): Promise<void> {
 	if (!select.exists('.reponav-dropdown')) {
 		createDropdown();
 	}
+	const insightsTab = select<HTMLAnchorElement>('[data-selected-links~="pulse"]')!;
+	const securityTab = select<HTMLAnchorElement>('[data-selected-links~="security"]')!;
 
 	select('.reponav-dropdown .dropdown-menu')!.append(
-		<a href={`/${repoUrl}/compare`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/compare`} className="rgh-reponav-more dropdown-item">
 			{icons.darkCompare()} Compare
 		</a>,
 
 		isEnterprise() ? '' :
-			<a href={`/${repoUrl}/network/dependencies`} className="rgh-reponav-more dropdown-item rgh-dependency-graph" data-skip-pjax>
+			<a href={`/${repoUrl}/network/dependencies`} className="rgh-reponav-more dropdown-item">
 				{icons.dependency()} Dependencies
 			</a>,
 
-		<a href={`/${repoUrl}/pulse`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
-			{icons.graph()} Insights
-		</a>,
-
-		<a href={`/${repoUrl}/commits`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/commits`} className="rgh-reponav-more dropdown-item">
 			{icons.history()} Commits
 		</a>,
 
-		<a href={`/${repoUrl}/branches`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/branches`} className="rgh-reponav-more dropdown-item">
 			{icons.branch()} Branches
 		</a>,
+
+		<a href={insightsTab.href} className="rgh-reponav-more dropdown-item">
```

Comment:
Can we only add the link if it exists? Confident GHE doesn't have this or it's configuration based on allowing it.

Reply:
We've been showing it without conditions for the longest time, but now I guess if it's not there the whole feature will fail.